storagenode/monitor/monitor.go

```diff
@@ -34,6 +34,8 @@ type Config struct {
 }
 
 // Service which monitors disk usage and updates kademlia network as necessary.
+//
+// architecture: Service
```

Comment:
I'm not clear why this is a service, could you explain to me, please?

Reply:
Yeah, this is one of those, where I'm completely unsure what it should be. It might be that this needs to be split into a service and a chore.